### DISCUSSION TITLE
query returns null values sometimes

### DISCUSSION THREAD

**Discussion by tg-sbeelaard** [webonyx/graphql-php#892] [Q&A]
I'm using Symfony 4.4, latest version of webonyx graphql-php and a pretty simple GraphQL query to get a company ID and company name.
The company ID is successfully retrieved but the `name` is not. Instead it's returning `null`, even when there is a name present in database.

Anyone who knows why it's returning null for the name? If you need more info please let me know.

I'm using `ApiPlatform\Core\GraphQl\Action\EntrypointAction` like this:

![image](https://user-images.githubusercontent.com/87526937/125939298-6aa558fe-f950-4a78-9b85-34c2943d9cd1.png)

![image](https://user-images.githubusercontent.com/87526937/125939455-fc75348f-333c-46dc-bda1-8e135034d8f0.png)

![image](https://user-images.githubusercontent.com/87526937/125937775-6ee0ab82-53f4-4f16-903e-dad13954c055.png)

**Comment by simPod** [comment]
IMO this is for ApiPlatform. Never used it tho.

Can you show Company class? Can you spot any differences between id and name properties/accessors?

**Reply by tg-sbeelaard** [reply]
Hi @simPod, thanks for your reply.

Here is the relevant part of the Company class. There is not much difference which can cause the earlier mentioned behaviour.



    /**
     * @Solr\Id
     * @ORM\Id
     * @ORM\Column(type="integer")
     * @ORM\GeneratedValue(strategy="AUTO")
     * @ApiProperty(identifier=false)
     */

    protected $id;

    /**
     * @ORM\Column(type="uuid_binary", unique=true, nullable=false)
     * @ORM\GeneratedValue(strategy="CUSTOM")
     * @ORM\CustomIdGenerator(class="Ramsey\Uuid\Doctrine\UuidGenerator")
     * @ApiProperty(identifier=true)
     */
    protected $uuid;


    /**
     * @ORM\Column(type="string", length=255, nullable=true)
     */
    protected $name;

**Reply by simPod** [reply]
Sorry, can't help with this. I don't have the whole context. E.g. how is your id of type integer when in the screenshot it's string?

![image](https://user-images.githubusercontent.com/327717/126126748-5c50d2c5-df65-4f0e-a250-fe6d875366d5.png)

Check that the `name` field is hydrated. Can't tell how you access it.

**Reply by tg-sbeelaard** [reply]
`id` is just a general graphql identifier which references to the complete uri in our api platform.

My feeling is that somewhere in the default resolver things go wrong, but I can't figure out what or where exactly.
It's hard to create a testcase for you because this is part of a big project, but I'm happy to provide more info when needed.
